gems/uc3-dmp-external-api/lib/uc3-dmp-external-api/client.rb

```diff
@@ -25,11 +25,12 @@ def call(url:, method: :get, body: '', basic_auth: {}, additional_headers: {}, t
         opts = _options(body:, basic_auth:, additional_headers:, timeout:, logger:)
         resp = HTTParty.send(method.to_sym, uri, opts)
 
-        unless [200, 201].include?(resp.code)
-          msg = "status: #{resp&.code}, body: #{resp&.body}"
-          raise ExternalApiError, "#{format(MSG_ERROR_FROM_EXTERNAL_API, url:)} - #{msg}"
+        if [200, 201].include?(resp.code.to_i)
+          resp.body.nil? || resp.body.empty? ? nil : _process_response(resp:)
```

Comment:
Rails returns the last line of the function unless you explicitly tell it to `return`, so change this statement inside the `if` so that it says: `return  resp.body.nil? || resp.body.empty? ? nil : _process_response(resp:)`

Reply:
Thanks for the info Brian. I made the update.